Module-3 | Projects with GitHub/controllers/commitController.js

```diff
@@ -0,0 +1,95 @@
+const Request = require("request");
+const commits = require('../models/commitModel');
+const commit = require('../models/commitModel');
+const repos = require('../models/repoModel');
+const Sequelize = require('sequelize');
+const bodyParser = require('body-parser');
+const _ = require('lodash');
+
+exports.commitList = (req,res) => {
+  commits.findAll().then(commits => {
+    res.send(commits);
+  });
+}
+
+exports.getCommitList = (req,res) => {  
+  commits.findAll()
+   .then((commits) => requestGitHub(commits,req,res));
+}
+
+const requestGitHub = (commits,req,res) => {
```

Comment:
instead of writing code to interact with Github everywhere, make a separate GuthubService for it. Export methods from that service and use those methods here.